src/Uecode/Bundle/ApiKeyBundle/Security/Firewall/ApiKeyListener.php

```diff
@@ -17,7 +17,7 @@
 class ApiKeyListener implements ListenerInterface
 {
     /**
-     * @var SecurityContextInterface
+     * @var TokenStorageInterface
      */
     protected $securityContext;
```

Comment:
done, but is it safe for BC? Field is protected so it can be used by class childs.
May be make these fields private?